static/style.css

```diff
@@ -124,13 +127,41 @@ header h2 {
     border-top: 2px solid #262119ff;
     border-bottom: 2px solid #262119ff;
 }
+#hamburger {
+    display: none;
+}
+
 @media only screen and (max-width: 600px) {
-    nav ol {
+    #hamburger {
+        display:inline-block;
+        position: fixed;
+        top: 0px;
+        right: 2%;
+        z-index: 2;
+        padding: 4%;
+        border: solid 1px #53462bff;
+        border-radius: 30%;
+        font-size: 1.5rem;
+        font-weight: 800;
+        color: #53462bff;
+    }
+    #hamburger:hover {
+        cursor: pointer;
+    }

```

Comment:
(I'll often prefix comments in a PR with `nit:`. This means "this is a nitpick, and it's not a major thing, but a minor style thing that usually makes things a bit easier to read or more consistent". 

nit: Get comfy with the idea of adding a blank line of vertical space between different concepts to make things easier to read. 
```
 #hamburger:hover {
        cursor: pointer;
}

.navbar {
        display: none;
        position: fixed;
        top: 0px;
        right: 10%;
        z-index: 1;
        border-radius: 12%;
    }
```
is easier to quickly spot a chunk of code for each section than if it all strings together. It also gives you the option to omit the vertical space if 2 things are related and you want them to be treated as a "single chunk of related content". But honestly I err on the side of including it more often than not.